helm-chart/templates/nextcloud-aio-apache-service.yaml

```diff
@@ -0,0 +1,18 @@
+apiVersion: v1
+kind: Service
+metadata:
+  annotations:
+    kompose.cmd: kompose convert -c -f latest.yml
+    kompose.version: 1.27.0 (b0ed6a2c9)
+  labels:
+    io.kompose.service: nextcloud-aio-apache
+  name: nextcloud-aio-apache
+spec:
```

Comment:
sorry, what does this do?

Reply:
NodePort exposes the port on the host. ClusterIP for example only exposes inside the cluster. It could be useful for people running without it without an ingress.

Reply:
So a service is not enough to make this run on all different cluster types?
See https://github.com/nextcloud/all-in-one/blob/main/helm-chart/templates/nextcloud-aio-apache-service.yaml